src/main/java/com/czertainly/api/model/core/certificate/CertificateDto.java

```diff
@@ -136,4 +136,10 @@ public class CertificateDto {
             requiredMode = Schema.RequiredMode.REQUIRED
     )
     private boolean privateKeyAvailability;
+
+    @Schema(
+            description = "Indicator whether CA is marked as trusted",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private boolean trustedCa;
```

Comment:
Change to wrapper type `Boolean` and add to description what it means if it is `null` - certificate is not CA certificate